lib/fluent/plugin/in_beats.rb

```diff
@@ -42,17 +42,22 @@ def initialize
     config_param :ssl_key, :string, :default => nil
     config_param :ssl_key_passphrase, :string, :default => nil
 
+    config_section :parse do
+      config_set_default :@type, DEFAULT_PARSER
```

Comment:
By default, beats plugin doesn't parse the message because this feature is for filebeat.
The default should be `nil` for such case?

Reply:
It seems that parser_create cannot hanle `nil` but it can handle `none` for this case.

Reply:
How about don't call `parser_create` when `<parse>` is not configured?

Reply:
I've avoided to call `parser_create` when `<parse>` is not configured.